include/envoy/thread_local/thread_local.h

```diff
@@ -74,6 +74,16 @@ class Slot {
    */
   using InitializeCb = std::function<ThreadLocalObjectSharedPtr(Event::Dispatcher& dispatcher)>;
   virtual void set(InitializeCb cb) PURE;
+
+  // UpdateCb takes the current stored data, and returns an updated/new version data.
```

Comment:
nit: Convert to `/**` comments for consistency.

Reply:
done